_functions.sh

```diff
@@ -1469,6 +1507,9 @@ do_undeploy() {
     if ${DEPLOYMENT_DATABASE_ENABLED}; then
       do_drop_database
     fi
+    if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
```

Comment:
Same remark as previously, this test should be moved into the do_start_onlyoffice method

Reply:
the test was moved to do_start_onlyoffice()